plugins/modules/zabbix_maintenance.py

```diff
@@ -350,12 +350,12 @@ def main():
             module.fail_json(msg="Failed to check maintenance %s existence: %s" % (name, error))
 
         if maintenance and (
-            sorted(group_ids) != sorted(maintenance["groupids"]) or
-            sorted(host_ids) != sorted(maintenance["hostids"]) or
-            str(maintenance_type) != maintenance["maintenance_type"] or
-            str(int(start_time)) != maintenance["active_since"] or
-            str(int(start_time + period)) != maintenance["active_till"] or
-            str(desc) != maintenance['description']
+            sorted(group_ids) != sorted(maintenance["groupids"])
+            or sorted(host_ids) != sorted(maintenance["hostids"])
+            or str(maintenance_type) != maintenance["maintenance_type"]
+            or str(int(start_time)) != maintenance["active_since"]
+            or str(int(start_time + period)) != maintenance["active_till"]
+            or str(desc) != maintenance['description']
```

Comment:
isn't this against PEP8 or something?

Reply:
W503 is currently deprecated.
see: https://www.flake8rules.com/rules/W503.html

W504 is currently recommended.
see: https://www.flake8rules.com/rules/W504.html